.eslintrc.js

```diff
@@ -13,4 +13,5 @@ module.exports = {
       },
     },
   ],
+  ignorePatterns: ['README.md', 'tests/fixtures'],
```

Comment:
Any reason to exclude `README.md` here? Linting the README is intentional (it's part of our [Node project template](https://github.com/netlify/node-template/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/package.json#L27)), and it allows us to not only have a consistent Markdown syntax, but also ensure that any code samples within it are also following our conventions.

Reply:
fixed with 1bfdfeb